cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return f"{self.first_name}{self.last_name}"
```

Comment:
The `__str__` method for the `Actor` model should include a space between the first and last names for better readability. Consider changing it to `return f"{self.first_name} {self.last_name}"`.